src/components/schemas/reimbursement.yaml

```diff
@@ -13,22 +13,64 @@ reimbursement_out:
       $ref: './fields.yaml#/created_at'
     updated_at:
       $ref: './fields.yaml#/updated_at'
+    
+    reimbursement_number:
+      type: string
+      description: |
+        A unique, organization wide identifier for a particular reimbursement.<br>
+        This id is always prefixed with `P/`.
+      example: P/2020/11/29
+
+    creator_user_id:
```

Comment:
do we need creator_user_id ? 
we have created_by and updated_by

Reply:
hmm, couldn't find any mention of these fields in any OpenAPI spec so assumed they might not be external. are they ?